recipe/build.sh

```diff
@@ -83,3 +84,26 @@ for links in ${SRC_DIR}/build/qt*/user_facing_tool_links.txt
 do
   while read _line; do ln -sf $_line; done < ${links}
 done
+
+# You can find the expected values of these files in the log
+# For example Translations will be listed as
+# INSTALL_TRANSLATIONSDIR
+# This file should be in the location of the user's executable
+# for conda, this becomes PREFIX/bin/
+# https://doc.qt.io/qt-6/qt-conf.html
+cat << EOF >${PREFIX}/bin/qt6.conf
```

Comment:
I tested this by forcing my application to use PySide6, without this:

1. Install pyside2 and pyside6 (thus installing qt-main and qt6-main in the process)
2. Try to force your application to start in PySide `QT_API=PySide6` and ensure you `import PySide6` before `qtpy` or anything else

<details><summary>error</summary>

```
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqoffscreen.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqminimalegl.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqminimal.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqeglfs.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqxcb.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqwebgl.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqwayland-egl.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqwayland-xcomposite-glx.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqvnc.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqwayland-generic.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.core.plugin.loader: In /home/mark/mambaforge/envs/mcam_dev/plugins/platforms/libqwayland-xcomposite-egl.so:
  Plugin uses incompatible Qt library (5.15.0) [release]
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```
</details>


Then executing:
```
export PREFIX=${CONDA_PREFIX}
```
and the rest of the cat command here, my application was able to start correctly.

It complained it couldn't find the wayland plugin, but at least it was able to load the rest of the stuff:
```
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
```

Reply:
Thank you for testing. A PR would be greatly appreciated 


I'm not really sure the cmake command is easier than the custom script. We would have to make a cake project and invoke it. Seems tedious.

 I had a few typos. But I don't think it will chnage over time so the maintenance will be low.

If you make a PR it can be pushed forward quickly.

Reply:
See PR #134